src/send-games-data/send-games-info.controller.ts

```diff
@@ -6,16 +6,8 @@ import { sendGameData } from './utils/send-game-data';
 export class SendGamesInfoController {
   @Get()
   async getGameData(@Res() res: Response, @Query('title') title?: string) {
-    const gameData = await sendGameData();
+    const gameData = await sendGameData(title);
```

Comment:
THIS function is not SENDING DATA it's fetching it from file. WHY THE NAME DOES NOT REPRESENT ITS FUNCTION??

Reply:
i go to HR